helm/oauth2-proxy/README.md

```diff
@@ -117,6 +117,16 @@ Parameter | Description | Default
 `securityContext.enabled` | enable Kubernetes security context on container | `false`
 `securityContext.runAsNonRoot` | make sure that the container runs as a non-root user | `true`
 `proxyVarsAsSecrets` | choose between environment values or secrets for setting up OAUTH2_PROXY variables. When set to false, remember to add the variables OAUTH2_PROXY_CLIENT_ID, OAUTH2_PROXY_CLIENT_SECRET, OAUTH2_PROXY_COOKIE_SECRET in extraEnv | `true`
+`sessionStorage.cookie.enabled` | enable cookie based session storage | `true`
+`sessionStorage.redis.enabled` | enable redis session storage | `false`
```

Comment:
Since the type of storage is an enum in OAuth2 Proxy, could we do the same here, ie, change these two fields to a `type` field that allows a string input?